app_dart/lib/src/request_handlers/refresh_github_commits.dart

```diff
@@ -218,7 +218,7 @@ class RefreshGithubCommits extends ApiRequestHandler<Body> {
       newTask(
           'mac_bot', 'chromebot', <String>['can-update-chromebots'], false, 0),
       newTask('linux_bot', 'chromebot', <String>['can-update-chromebots'],
-          false, 0),
+          true, 0),
```

Comment:
I'm not really sure if this will work, let's see!

Reply:
luci bots follow a different workflow but let's try it. If this does not work I'll disable the test.